### PR TITLE
Add terra staked balances to TBC.

### PR DESCRIPTION
**Pull request by jnaviask**
NEEDS QA.

Outstanding TODOs (maybe not necessary): pagination on query.

We can also remove the key stuff and use the phoenix-lcd endpoint rather than the setten one as the production endpoint.

### CODE REVIEW

**Review by zakhap — COMMENTED** [review]
seems good to me on glance, would love to talk through robustly QAing/testing this. I think we should probably still use the key'd phoenix endpoint tho right? Do we know how/much/when we'd be rate limited by the public endpoint?

File: packages/token-balance-cache/src/provider.ts [hunk 1]
```diff
@@ -118,21 +121,46 @@ export default class TokenBalanceProvider {
...
-      throw new Error(`no balance found: ${e.message}`);
+      log.warn(`could not fetch terra bank balance: ${e.message}`);
     }
+
+    // make staking query
+    let stakedBalance = new BN(0);
+    const stakedQueryUrl = `${url}/cosmos/staking/v1beta1/delegations/${
+      userAddress
+    }?key=${
+      process.env.TERRA_SETTEN_PHOENIX_API_KEY
```
Comment (zakhap) [inline packages/token-balance-cache/src/provider.ts:144]: let's remove this and use the lcd phoenix url, as terra says it should be fine for prod 🤷

**Comment by jnaviask** [comment]
Should be good to go. Tested and it works, computes correctly.

**Review by zakhap — APPROVED** [review]
lgtm 🚀

**Pull request merged**